autoscale_cloudcafe/autoscale/client.py

```diff
@@ -870,3 +902,86 @@ def delete_node(self, load_balancer_id, node_id, requestslib_kwargs=None):
                              str(node_id)])
         return self.request('DELETE', full_url,
                             requestslib_kwargs=requestslib_kwargs)
+
+
+class RackConnectV3APIClient(AutoMarshallingRestClient):
+    """
+    Client objects for all Rackconnect V3 API calls.
+    """
+
+    def __init__(self, url, auth_token, serialize_format=None,
+                 deserialize_format=None):
+        super(RackConnectV3APIClient, self).__init__(serialize_format,
+                                                     deserialize_format)
+        self.url = url
+        self.auth_token = auth_token
+        self.default_headers['X-Auth-Token'] = auth_token
+        self.default_headers['Content-Type'] = (
+            'application/%s' % (self.serialize_format,))
+        self.default_headers['Accept'] = 'application/%s' % (
+            (self.deserialize_format,))
+
+    def list_pools(self):
+        """
+        :summary: Get the list of pools for the tenant's RCV3 account
+        :return: Response Object containing response code 202
+        on success and list of pools
+        :rtype: Response Object
+        """
+        url = self.url + "/load_balancer_pools"
+        return self.request('GET', url,
+                            response_entity_type=RackConnectLBPools)
+
+    def get_cloud_network_info(self, cloud_network_id):
+        """
+        :summary: Get information on a given cloud_network
+        :return: Response Object
+        :rtype: Response Object
+        """
+        url = self.url + '/cloud_networks/{0}'.format(cloud_network_id)
+        print("### GETTING NETWORK INFO FROM: " + url)
+        return self.request('GET', url,
+                            response_entity_type=RackConnectNetworkInfo)
+
+    def get_pool_info(self, pool_id):
+        """
+        :summary: Get information on a given load_balancer_pool
+        :return: Response Object
+        :rtype: Response Object
+        """
+        url = self.url + '/load_balancer_pools/{0}'.format(pool_id)
+        print("... RCV3 request ... ", url)
```

Comment:
Would it make sense to remove this print statement and the three below it?  These seem more like debug print statements than actual useful information in the log outputs (partially because they don't convey a lot of information other than the URLs, and because these calls seem to be made over and over and over in a loop, and seeing that it's polling RCv3 and alone is not necessarily helpful in reading test output).